### PR TITLE
Adding Wose Shaman animations to mainline

### PR DESCRIPTION
**Pull request by Earth-Cake**
There are some existing animation sprites for Wose Shaman, used in WoV and DM, so basically this Pull Request is made to get these sprites in those campaigns. I have also updated unit files that they match sprites and corrected sound errors.

### CODE REVIEW

File: data/campaigns/Delfadors_Memoirs/units/Wose_Shaman.cfg [hunk 1]
```diff
@@ -4,15 +4,6 @@
     name=_ "Wose Shaman"
     race=wose
     image=units/wose-shaman.png
-    [standing_anim]
-        start_time=0
-        [frame]
-            image="units/wose-shaman.png"
-            halo="halo/wose-stationary-halo[1~8].png:150"
-        [/frame]
-    [/standing_anim]
```
Comment (stevecotton) [inline data/campaigns/Delfadors_Memoirs/units/Wose_Shaman.cfg]: Is the standing_anim deliberately removed?

Comment (Earth-Cake) [reply data/campaigns/Delfadors_Memoirs/units/Wose_Shaman.cfg]: Mainline Woses don't have this animation, and there are no animation sprites to support this tags so I didn't find it appropriate to keep this anim.

Comment (Earth-Cake) [reply data/campaigns/Delfadors_Memoirs/units/Wose_Shaman.cfg]: @stevecotton I added them.

**Comment by stevecotton** [comment]
@sigurdfdragon as this would touch your campaigns, please would you review?

There's a follow-up PR #4226 which would move these graphics to core.

**Comment by soliton-** [comment]
Wait, why would we first replace the same graphics multitple times only to _then_ move them to core? It really should be the other way around. We don't need to bloat the repository unnecessarily.

**Comment by jostephd** [comment]
@soliton- What bloat are you talking about? Git will store the png just once regardless of how many times it appears in the tree, and the worktree size won't be meaningfully affected (the image files are 8KB each, my worktrees are 800MB each). Developer time is more important to optimize for.

**Comment by stevecotton** [comment]
The reason to do #4226 after this one is because I'm more familiar with Git than EarthCake is. But I'd happily squash both PRs into a single commit.

**Comment by soliton-** [comment]
Well, if git optimizes this perfectly then that's great. No idea how this optimizes developer time though. I would have already merged #4226 if it wasn't closed again. Perhaps I'm biased because I cannot help approve the images.

**Comment by Earth-Cake** [comment]
> The reason to do #4226 after this one is because I'm more familiar with Git than EarthCake is. But I'd happily squash both PRs into a single commit.

I must say that you are right xD

**Comment by stevecotton** [comment]
> Well, if git optimizes this perfectly then that's great. No idea how this optimizes developer time though. I would have already merged #4226 if it wasn't closed again. Perhaps I'm biased because I cannot help approve the images.

If we're waiting to approve the images then it's quite easy to swap the PRs the other way round again.

**Comment by sigurdfdragon** [comment]
I'm fine with the animation changes and the wose shaman being moved to core

**Comment by sevu** [comment]
Merging then for 1.15.1.

How did you swap the two PRs?

**Comment by stevecotton** [comment]
> How did you swap the two PRs?

`git reset --soft`, and commit the result.

Start with two separate commits:

* *A*  with just the animations
* *C* with just the move to core

At one stage the idea was that A would go first, so I created C' on top of A:

origin/master - A - C'

For simplicity assume that nothing else is being committed to origin/master in the meantime. Then we decided to merge C first, so I took my old pre-rebase version of C:

origin/master - C

Finally we need an A' that's on top of C. The tree (files on the filesystem) for A' is exactly the same as those for C', so A' can be created by:

1. create a branch on commit C'
1. `git reset --soft` to C
1. commit the result, with the commit message saying that it's A'

**Pull request merged**